### DISCUSSION TITLE
trice on win example

### DISCUSSION THREAD

**Discussion by MCO-SOCOMEC** [rokath/trice#272] [General]
Hello,
I read about this interesting project. Is there any port example on windows? To try it I was thiking to a small c project sending trice info over tcp/IP collected back by the logger.
thanks
michele

**Comment by rokath** [comment]
The MDK... projects inside the test folder are built in Windows. In the releases folder unzip trice.exe for Windows. The C code is OS agnostic. So set up a target project with the compiler of your choice and add trice.c as part of it.

**Reply by rokath** [reply]
replied

**Reply by MCO-SOCOMEC** [reply]
Thank you, I will try it as soon as I go back in office. Thanks
